cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -114,11 +114,18 @@ def pytest_generate_tests(metafunc):
 
 
 @pytest.fixture(scope="module")
-def vm_analysis_provisioning_data(provider, analysis_type):
-    vma_data = provider.data.vm_analysis_new
+def ssa_provider(provider):
+    "Create customized provider on CFME"
+    provider.create_rest(check_existing=True)
+    return provider
+
+
+@pytest.fixture(scope="module")
+def vm_analysis_provisioning_data(ssa_provider, analysis_type):
+    vma_data = ssa_provider.data.vm_analysis_new
     provisioning_data = vma_data.provisioning
 
-    if not isinstance(provider, CloudProvider):
+    if not isinstance(ssa_provider, CloudProvider):
```

Comment:
Required:
I know its right, but old style now we have `one_of` method for provider please use it like
`if not ssa_provider.one_of(CloudProvider):`

Reply:
Sure will add it.